### PR TITLE
add book and remove book functionality added

### PR DESCRIPTION
**Pull request by kelibst**
Create a collection that keeps a list of books (hint: you can use an array of objects for that).
Create a function to add a new book to the collection, with title and author.
Create a function to remove a book from the collection (hint: you can use the array filter() method).
Display all books saved in the collection in the top part of the page.
Make sure that when a user clicks the "Add" button:
A new book is added to the collection.
The new book is displayed in the page.
Make sure that when a user clicks the "Remove" button:
The correct book is removed from the collection.
The correct book dissapears from the page.
Make sure that data is preserved in the browser's memory by using localStorage.
To debug your application, you can use console.log(). Remember to remove debugging code before pushing your code to the remote repository.
Don't use any library or framework, use only plain JavaScript.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @kelibst 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/scripts/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+function showBooks() {
+  let book = '';
+  if (books.length === 0) {
+    bookList.innerHTML = '<p>Sorry you have no book left. Kindly add some</p>';
+  } else {
+    books.forEach((bookObj, ind) => {
+      book += `<li class="book-li">
+    <span>${bookObj.name}</span> <span>${bookObj.title}</span>
```
Comment (mikenath223) [inline src/scripts/index.js]: - [ ] Kindly change these functions to es6 arrow function syntax.

File: src/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+li.nav-li {
+  padding: 0 10px;
+  background: white;
+}
+
+a {
+  text-decoration: none;
+
+  /* font-size: 18px; */
```
Comment (mikenath223) [inline src/styles/style.css]: - [ ] Please remove this commented code.

File: src/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+  float: right;
+  margin-right: 2rem;
+  font-size: 1.2rem;
+  font-family: sans-serif;
+  font-weight: bold;
+}
+
+/* #form {
+    display: none;
+  } */
```
Comment (mikenath223) [inline src/styles/style.css]: - [ ] Please remove this commented code.

**Review by dasileker — APPROVED** [review]
## **Hi @kelibst,** 

### **Nie Job👌**

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**